packages/allure-playwright/README.md

```diff
@@ -45,6 +45,14 @@ set ALLURE_RESULTS_DIR=my-allure-results
 npx playwright test --reporter=line,allure-playwright
 ```
 
+or inside the config via:
+
+```js
+const config = {
+  reporter: [ ['allure-playwright', { outputFolder: 'my-allure-results' }] ],
```

Comment:
maybe align the name with other modules? As far as I know, `resultsDir` is used in there.

Reply:
Inside the `@playwright/test` world `outputDir` and `outputFile` seems to be the norm: https://playwright.dev/docs/test-reporters#html-reporter, are you fine with aligning it that way?